js/gdm/authPrompt.js

```diff
@@ -660,7 +660,26 @@ const AuthPrompt = new Lang.Class({
     },
 
     _computeUnlockCode: function(resetCode) {
+        const PASSWORD_RESET_GROUP_NAME = 'Password Reset';
+        const PASSWORD_RESET_KEY_SALT = 'Salt';
+
+        let keyFile = this._loadCustomerSupportFile(),
+            salt = '';
```

Comment:
Please use `let` here as well (don't combine declaration of 2 variables under the same `let`)

Reply:
Still pending to address